config/prow/cluster/monitoring/alertmanager-prow_secret.yaml

```diff
@@ -22,8 +22,14 @@ stringData:
       - receiver: 'slack-alerts'
         group_interval: 5m
         repeat_interval: 2h
-        match_re:
-          severity: 'critical|high'
+        matchers:
+          - boskos_type != aws-account
+          - severity =~ critical|high
+      - receiver: 'cluster-api-aws-alerts'
+        group_interval: 5m
+        repeat_interval: 2h
+        matchers:
+          - boskos_type = aws-account
```

Comment:
Listing this route before the route for the `slack-alerts` receiver would simplify this and make it more clear. Only the first route that matches will be used unless `continue` is set to `true`. See the [docs for routes](https://prometheus.io/docs/alerting/latest/configuration/#route). This would also allow us to keep the main alerting route the same so we'll be less likely to break existing alerts.

Reply:
Cool, i'll update.

Reply:
Updated